docs/modules/patterns/pages/communicate-facts.adoc

```diff
@@ -23,10 +23,10 @@ _Choose immutable event streams over mutable state_
 
 Instead, rely on immutable state—values representing _facts_—which can be shared safely as local or distributed _events_ without worrying about corrupt or inconsistent data, or guarding it with transactions or locks. 
 
 A fact is immutable and represents something that has already happened sometime in the past, something that can not be changed or retracted. It is a stable value that you can reason about and trust, indefinitely. After all, we can’t change the past, even if we sometimes wish that we could. Knowledge is cumulative and occurs either by receiving new facts or by deriving new facts from existing facts. Invalidation of existing knowledge is done by adding new facts to the system that refute existing facts. Facts are never deleted, only made irrelevant for current knowledge.
 
-Facts are best shared by publishing them as _events_ through the component’s _event stream_ where they can be subscribed to and consumed by others—components, databases, or subsystems—serving as a medium for communication, integration, and replication. _Facts_ stored as _events_ in an https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying[_event log] https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying[{tab}, window="new tab"], in their causal order, can represent the full history of a component’s state changes through time (e.g. using the https://martinfowler.com/eaaDev/EventSourcing.html[Event Sourcing] https://martinfowler.com/eaaDev/EventSourcing.html[{tab}, window="new tab"] pattern) while serving reads safely from memory (called https://martinfowler.com/bliki/MemoryImage.html[Memory Image] https://martinfowler.com/bliki/MemoryImage.html[{tab}, window="new tab"]).
\ No newline at end of file
+Facts are best shared by publishing them as _events_ through the component’s _event stream_ where they can be subscribed to and consumed by others—components, databases, or subsystems—serving as a medium for communication, integration, and replication. _Facts_ stored as _events_ in an https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying[_event log] https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying[{tab}, window="new tab"], in their causal order, can represent the full history of a component’s state changes through time (e.g. using the https://martinfowler.com/eaaDev/EventSourcing.html[Event Sourcing] https://martinfowler.com/eaaDev/EventSourcing.html[{tab}, window="new tab"] pattern) while serving reads safely from memory (called https://martinfowler.com/bliki/MemoryImage.html[Memory Image] https://martinfowler.com/bliki/MemoryImage.html[{tab}, window="new tab"]).

```

Comment:
I can’t find a change here, but it looks like there is a `_` missing at the end of the “event log” link, could you fix that as well?

Reply:
I originally edited this in the github web editor, which added a newline, but happy to make that change, too.